napari/layers/points/points.py

```diff
@@ -1260,6 +1260,26 @@ def _get_value(self, position) -> Union[None, int]:
 
         return selection
 
+    def _get_value_3d(self, **kwargs):
+        """Get the layer data value along a ray. Currently, this just
+        returns None. We will implement this in future work.
+
+        Parameters
+        ----------
+        start_position : np.ndarray
+            The start position of the ray used to interrogate the data.
+        end_position : np.ndarray
+            The end position of the ray used to interrogate the data.
+        dims_displayed : List[int]
+            The indices of the dimensions currently displayed in the Viewer.
+
+        Returns
+        -------
+        value
+            The data value along the supplied ray.
+        """
+        return None
```

Comment:
I vaguely prefer for the base layer to do this and then not create a bunch of None implementations...?

Reply:
This makes sense to me. Just to make sure we're on the same page, you're suggesting that we make an implementation like this of _get_value_3d on the base layer that returns None and then we override it on the layers we have real implementations. Is that right?